.github/CONTRIBUTING.md

```diff
@@ -13,27 +13,26 @@ project, you agree to abide by its terms.
 
 ## General workflow
 
-This is the workflow for contributing changes to Emogrifier:
+This is the workflow for contributing changes to this project::
 
-1. [Fork the Emogrifier Git repository](https://guides.github.com/activities/forking/).
-2. Clone your forked repository and
+1. [Fork the Git repository](https://docs.github.com/en/get-started/exploring-projects-on-github/contributing-to-a-project).
+2. Clone your forked repository locally and
    [install the development dependencies](#install-the-development-dependencies).
-3. Add a local remote "upstream" so you will be able to
-   [synchronize your fork with the original Emogrifier repository](https://help.github.com/articles/syncing-a-fork/).
-4. Create a local branch for your changes.
-5. [Add unit tests for your changes](#unit-test-your-changes).
+3. Create a local branch for your changes.
+4. [Add unit tests for your changes](#unit-test-your-changes).
    These tests should fail without your changes.
-6. Add your changes. Your added unit tests now should pass, and no other tests
+5. Add your changes. Your added unit tests now should pass, and no other tests
    should be broken. Check that your changes follow the same
    [coding style](#coding-style) as the rest of the project.
-7. Add a changelog entry.
-8. [Commit](#git-commits) and push your changes.
-9. [Create a pull request](https://help.github.com/articles/about-pull-requests/)
-   for your changes. Check that the Travis build is green. (If it is not, fix
-   the problems listed by Travis.)
-10. [Request a review](https://help.github.com/articles/about-pull-request-reviews/)
-    from @oliverklee.
-11. Together with him, polish your changes until they are ready to be merged.
+6. Add a changelog entry, newest on top.
+7. [Commit](#git-commits) and push your changes.
+8. [Create a pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests)
+   for your changes.
+9. Check that the CI build is green. (If it is not, fix the problems listed.)
+   Please note that for first-time contributors, you will need to wait for a
+   maintainer to approve your pull request before the CI build will run.
```

Comment:
Sorry to be nitpicky, but this implies that a maintainer would have approved the changes, whereas all they've done is clicked a button to allow the CI build to run.

```suggestion
   maintainer to allow your CI build to run.
```